cmd/ncproxy/ncproxy.go

```diff
@@ -557,6 +558,58 @@ func (s *grpcService) GetNetworks(ctx context.Context, req *ncproxygrpc.GetNetwo
 // client connections.
 type ttrpcService struct {
 	m sync.Mutex
+	// database for containerID to compute agent address
+	agentStore *computeAgentStore
+}
+
+func newTTRPCService(ctx context.Context, db *bolt.DB) *ttrpcService {
+	agentStore := newComputeAgentStore(db)
+	return &ttrpcService{
+		agentStore: agentStore,
+	}
+}
+
+func getComputeAgentClient(agentAddr string) (computeagent.ComputeAgentService, error) {
+	conn, err := winio.DialPipe(agentAddr, nil)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to connect to compute agent service")
+	}
+	client := ttrpc.NewClient(
+		conn,
+		ttrpc.WithUnaryClientInterceptor(octtrpc.ClientInterceptor()),
+		ttrpc.WithOnClose(func() { conn.Close() }),
+	)
+	return computeagent.NewComputeAgentClient(client), nil
```

Comment:
This looks like another case where we may be leaking `conn`. Since we return only a `ComputeAgentService`, which doesn't have a `Close` method, it seems like nothing will ever clean this up.

Reply:
This has been addressed